docs/agent.rst

```diff
@@ -48,54 +48,50 @@ Run an Agent
+* ``--baudrate <baudrate>``: set the baud rate of the communication. It can take the following values:
+  0, 50, 75, 110, 134, 150, 200, 300, 600, 1200, 1800, 240, 4800, 9600, 19200, 38400, 57600, 115200 (default), 230400, 460800, 500000, 576000, 921600, 1000000, 1152000, 1500000, 2000000, 2500000, 3000000, 3500000 or 4000000 Bd.
+* ``--refs <refs-file>``: load a reference file using the relative path to the working directory.
+  This reference file shall be composed by a set of Fast RTPS profile following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ describes in Fast RTPS.
+  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity, so it could be used by the *Clients* in order to create entities by reference.
 
-For running the *Agent* is necessary that the file ``DEFAULT_FASTRTPS_PROFILES.xml`` is located in the runtime folder.
-This file contains XML profiles of `Participants`, `Topic`, `DataWriters` and `DataReaders` referenced by a ``profile_name``.
-The *Client* can use the aforementioned ``profile_name`` in order to create these entities using the reference representation.
-Let's see an example.
+**Pseudo-Serial communication** (only Linux) ::
 
-The ``DEFAULT_FASTRTPS_PROFILES.xml`` contains the following participant profile:
-
-.. code-block:: xml
-
-    <profiles>
-      <participant profile_name="default_xrce_participant_profile">
-        <rtps>
-          <name>default_participant</name>
-        </rtps>
-      </participant>
-    </profile>
-
-therefore, the *Client* can use `default_xrce_participant_profile` as ``ref`` in the ``mr_buffer_create_participant_ref`` function.
-
-Default installation place this file along with the *Agent* executable into ``/usr/local/bin`` for Linux or ``C:/Program Files/microxrcedds_agent/bin`` for Windows.
+    $ ./MicroXRCEAgent --pseudo-serial [--baudrate <baudrate>] [--refs <refs-file>]
 
+Launch a pseudo-serial server which provide a device to which *Clients* can connect. It has the following options:
 
+* ``--baudrate <baudrate>``: set the baud rate of the communication. It can take the following values:
+  0, 50, 75, 110, 134, 150, 200, 300, 600, 1200, 1800, 240, 4800, 9600, 19200, 38400, 57600, 115200 (default), 230400, 460800, 500000, 576000, 921600, 1000000, 1152000, 1500000, 2000000, 2500000, 3000000, 3500000 or 4000000 Bd.
+* ``--refs <refs-file>``: load a reference file using the relative path to the working directory.
+  This reference file shall be composed by a set of Fast RTPS profile following the `XML syntax <https://eprosima-fast-rtps.readthedocs.io/en/latest/xmlprofiles.html>`_ describes in Fast RTPS.
+  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity, so it could be used by the *Clients* in order to create entities by reference.
```

Comment:
```suggestion
  The ``profile_name`` attribute of each profile represents a reference for an XRCE-Entity so that it could be used by the *Clients* to create entities by reference.
```

Reply:
Done!